### PR TITLE
Add fastlane match and update CircleCI info

### PR DESCRIPTION
**Pull request by justinswart**
This updates our docs to reflect the things that have changed in https://github.com/kickstarter/ios-oss/pull/331

### CODE REVIEW

File: ios-deployment.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ Run the command `make deploy` to deploy `private/master` to Hockey.
 ### App Store Deployment
 
 1. Bump the version number in Xcode and commit (build number is set automatically).
-2. Deploy `private/master` to iTunes connect by running the following command:
+1. Deploy `private/master` to iTunes connect by running the following command:
```
Comment (dusi) [inline ios-deployment.md]: Is this the right numba? 🤔

File: ios-deployment.md [hunk 2]
```diff
@@ -24,33 +24,26 @@ CircleCI, our CI provider, often updates Xcode and that breaks our builds and de
...
-```ruby
-before_all do
-  xcversion(version: "9.2")
-end
-```
+We use `fastlane match` to manage our certificates and provisioning profiles. CircleCI 2.0 also requires that we use `match`. All that you need to do is run `bundle exec fastlane match_all` in the project folder, however if you have not run this before then follow these steps:
 
-### iTunes connect
+1. run `bundle install`
+1. run `bundle exec fastlane match_all`
```
Comment (dusi) [inline ios-deployment.md]: Same here..looks like the list is not numbered properly 🤷‍♂️

**Comment by justinswart** [comment]
@dusi Thanks for reviewing, it will be numbered correctly when it's rendered on GitHub, I'm not sure if that's a markdown thing or a GitHub thing but I've usually just done that for numbered lists and then it figures it out 🙂see: https://github.com/kickstarter/native-docs/blob/f4579803304bdbc3663ebda6384c5eeb8032220f/ios-deployment.md

**Comment by dusi** [comment]
screen shot 2018-04-23 at 9 12 22 am

**Pull request merged**